### ISSUE TITLE
Add internationalization

### ISSUE THREAD

**Issue by olielvewen** [olielvewen/StreamCap#4]
Hi,

The initial version of StreamCap is in English. Even the software is simple to use, everybody doesn't speak/want to use in English. 
Add not all but major languages will be a plus for the software.

Thanks a lot.

**Comment by monschiTee** [comment]
Hi @olielvewen 
is there a file where I simply can add the language (in my case german) or
do I have to go through all your files and change the language?

**Comment by olielvewen** [comment]
Hi monschiTee,

Well, even it was I 've not planned to do (the PyQt5 port is a priority  because PyQt4 (if you don't know) will be removed in Linux Distribution in 2020 (when ? at the end ?),  

However, why not. After all, Streamcap is a simple program and anyway, this port is just done for keeping this program living. And I could that quickly.

So, for the moment you have nothing to do. I must create a Manifest file and prepare all files (like a German file for you). Normally, I will try to do that either this week-end either in the next week.
I'll told you when all will be ready.

In addition, there are several ways to translate (like transiflex, webtranslate) but for this program the easiest way is to use Qt tools and especially QT Linguist. By cons, I do not remember if they are included with the installation of Qt Creator or Qt Designer. You must use the version 5 of the second. I've just checked on my system (Manjaro) and you'll find it under the name qt5-tools. its will install Qt Designer5, Qt Linguist5 and Qt Assistant. 

In addition, I will open a wiki page for those who want to translate a program in Qt that it is in PyQt5 or in Qt5 and C++.

Thanks a lot about the interest that you have for this tiny application.

**Comment by Aresiel** [comment]
@olielvewen I am willing to help translate into Swedish, could you alert me when the translation file is available to edit?

**Comment by olielvewen** [comment]
Hi,

@Aresiel 

I have created today a Sweden file for making translation. I let you tomorrow how to do that. 

@monschiTee 

I have all done and resolved all issues that I have had. And I am now ready.

---

Actually, I have a french translator who help me to be a ......beta tester, like it is the first time that I'm doing that publicly.

I 'll told you tomorrow how to process.

Thanks a lot guys.

**Comment by olielvewen** [comment]
Hi,

I've created another wiki page to explain how to get a copy of a project (here streamcap) for making translation, documentation, code,....
This wiki page is [Participate to a project](https://github.com/olielvewen/StreamCap/wiki/Participate-to-a-project)

The last week I have created another wiki page about Translations. So read this [Translate a PyQt5 or Qt5 application](https://github.com/olielvewen/StreamCap/wiki/Translate-a-Pyqt5-or-QT5-application-(English-Version)).

---
**Note:**

If you don't understand something, ask me and I will update wiki 's pages.

Thanks again.
